code-studio/src/js/external/external.js

```diff
@@ -1,5 +1,3 @@
-//= require_self
-
```

Comment:
I assumed in this case it would tell sprockets where to put the source in this file relative to stuff imported through other sprockets directives.  By itself, it's basically a no-op.